src/app/Library/CrudPanel/Traits/Create.php

```diff
@@ -405,4 +369,18 @@ private function getRelationDetailsFromInput($input)
 
         return $relationDetails;
     }
+
+    protected function getFieldNamesArray($fields)
```

Comment:
I don't understand what this does. Does it return an array? Or does it only return fields with array names? Or does it get an array with the final input names? Please:
- [ ] choose a more specific name for the function, that explains what it does/returns
- [ ] add a docblock to explain what it does 
- [ ] (bonus: you can define an example in that docblock - input and output)